### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the form component.

The U.I snippet is attached below

image

### CODE REVIEW

File: Components/forms/forms.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+/* 
+.input-other
+{
+
+  height: 2.5rem;
+
+ 
+} */
```
Comment (yashpatel1797) [inline Components/forms/forms.css:84]: can remove comments before merging it.

**Review by yashpatel1797 — APPROVED** [review]
code looks good. just added a small sugesstion.

**Comment by JatuFaique** [comment]
Ui looks clean to me.

File: Components/forms/forms.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    border: 2px solid var(--default-color);
+    width: 80%;
+    height: 100%;
+}
+
+.text-field:focus {
+    border: 2px solid var(--primary-color);
+}
+
+/*
```
Comment (zabihshaik) [inline Components/forms/forms.css:77]: Better to remove comments.

File: Components/forms/forms.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+@media screen and (max-width:640px) {
+
+    .input-batch {
+
+        margin: 0.3rem;
+
+    }
+
+    .input-field {
+
```
Comment (zabihshaik) [inline Components/forms/forms.css:96]: Better to remove unnecessary blank spaces. Use Prettier to format the code.

File: Components/forms/forms.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="forms.css">
```
Comment (zabihshaik) [inline Components/forms/forms.html:8]: Better to add meta tag with your name in content

File: Components/forms/forms.html [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+    <title>Document</title>
```
Comment (zabihshaik) [inline Components/forms/forms.html:9]: Better to use the component name as title

**Review by zabihshaik — COMMENTED** [review]
Everything looks Good. Just a few suggestions in comments. Please check

File: Components/forms/forms.html (hunk 5 above)
Comment (raghav6202) [reply Components/forms/forms.html:9]: Thank You will look into it

**Pull request merged**